src/tweenable.rs

```diff
@@ -791,10 +791,19 @@ impl<T> Tweenable<T> for Tween<T> {
         // If completed at least once this frame, notify the user
         if times_completed > 0 {
             if let Some(user_data) = &self.event_data {
-                events.send(TweenCompleted {
+                let event = TweenCompleted {
                     entity,
                     user_data: *user_data,
-                });
+                };
+
+                // send regular event
+                events.send(event);
+
+                // trigger all global observers
+                commands.trigger(event);
+
+                // trigger all entity-scoped observers
+                commands.trigger_targets(event, [entity]);
```

Comment:
A small nit, but you don't need an array:
```suggestion
                commands.trigger_targets(event, entity);
```